src/UnityUtils.ts

```
@@ -44,6 +44,9 @@ export default class UnityUtils
         switch (os.platform()) {
         default:
             throw new Error('Unsupported platform.')
+        case 'linux':
+            const homeDir = os.homedir();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)